client/src/sockets/enums.js

```diff
@@ -47,7 +48,6 @@ export const ListEvents = Object.freeze({
   REMOVE_MEMBER_SUCCESS: ListActionTypes.REMOVE_MEMBER_SUCCESS,
   REMOVE_WHEN_COHORT_UNAVAILABLE: 'list/REMOVE_WHEN_COHORT_UNAVAILABLE',
   REMOVE_OWNER_ROLE_SUCCESS: ListActionTypes.REMOVE_OWNER_ROLE_SUCCESS,
-  REMOVED_BY_SOMEONE: 'list/REMOVED_BY_SOMEONE',
```

Comment:
this one is not needed anymore??

Reply:
jest it's not needed, all work is done in simple REMOVE_MEMBER_SUCCESS